tsl/test/sql/dist_util.sql.in

```diff
@@ -313,15 +313,10 @@ SELECT count(*) FROM size_test_table;
 SELECT 1 FROM pg_table_size('size_test_table');
 SELECT 1 FROM pg_table_size('disttable');
 SELECT 1 FROM pg_table_size('nondisttable');
-
--- hypertable_size requires SELECT privilege on table
-\set ON_ERROR_STOP 0
 SELECT * FROM hypertable_size('disttable');
 SELECT * FROM hypertable_size('nondisttable');
 SELECT * FROM hypertable_detailed_size('disttable') ORDER BY node_name;
 SELECT * FROM hypertable_detailed_size('nondisttable') ORDER BY node_name;
-\set ON_ERROR_STOP 1
-
```

Comment:
This seems like a weird thing to test at all. It's just an implementation aspect, not a key feature of the function, so why test it at all?